.azure-pipelines/win32-conda-CI.yml

```diff
@@ -62,6 +62,13 @@ jobs:
       pytest tests --doctest-modules --junitxml=junit/test-results.xml
     displayName: 'pytest'
 
+  - script: |
+      pip install coverage onnx
+      coverage run --include=keras2onnx/* --omit=keras2onnx/ktf2onnx/* tests/test_layers.py
```

Comment:
This command can be merged with pytest to run unit test and get the coverage at the same time. That might fix the build.